### PR TITLE
OSDOCS-6344: updates MTU parameter for InFw operator for 4.14

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-6344](https://issues.redhat.com//browse/OSDOCS-6344): updates MTU parameter for InFw operator for 4.14

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-6344

Link to docs preview:
https://60740--docspreview.netlify.app/openshift-enterprise/latest/networking/ingress-node-firewall-operator.html

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @msherif1234 
https://github.com/openshift/openshift-docs/pull/60836/files

### CODE REVIEW

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Comment by msherif1234** [comment]
@JoeAldinger while u are there pls indicate OCP 4.14 needs nodes with rhel9+ it won't work with older kernel, cc @anuragthehatter

File: modules/nw-infw-operator-cr.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ You configure `rules` of the `IngressNodeFirewall` CR and apply them to clusters
 ====
 The Ingress Node Firewall Operator supports only stateless firewall rules.
 
-The maximum transmission units (MTU) parameter is 4Kb (kilobytes) in {product-title} {product-version}.
-
 Network interface controllers (NICs) that do not support native XDP drivers will run at a lower performance.
+
+For {product-title} 4.14., you must run Ingress Node Firewall Operator on {op-system-base} 9.0 or later.
```
Comment (opayne1) [inline modules/nw-infw-operator-cr.adoc:19]:
```suggestion
For {product-title} 4.14, you must run Ingress Node Firewall Operator on {op-system-base} 9.0 or later.
```

**Review by opayne1 — COMMENTED** [review]
Noticed one tiny nit!

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**